corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js

```diff
@@ -854,6 +854,21 @@ cloudCare.AppMainView = Backbone.View.extend({
         self._navEnabled = true;
         self.router = new cloudCare.AppNavigation();
         self._sessionsEnabled = self.options.sessionsEnabled;
+        self._urlParams = {};
+
+        // http://stackoverflow.com/a/2880929/835696
```

Comment:
I'll assume that 1200 upvotes means this works fine so not thinking too hard about the regexes and what not